pkg/image/reference/reference.go

```diff
@@ -62,6 +63,19 @@ func Parse(spec string) (DockerImageReference, error) {
 	return ref, nil
 }
 
+// GetSpecDigest returns the image digest from a DockerImageReference string.
+// example 'sha:22204fa2...' in an image 'registry/ocp/release@sha:22204fa2...'
+func GetSpecDigest(spec string) (string, error) {
```

Comment:
Why this is needed when we can use `Parse` and then read resulting `ref.ID`?

Reply:
correct, closing this lol